src/eatery_db/collegetown_eatery.py

```diff
@@ -39,3 +45,43 @@ def parse_collegetown_eateries(collegetown_data):
         collegetown_eateries.append(new_eatery)
 
     return collegetown_eateries
+
+
+def parse_collegetown_hours(data_json, eatery_model):
```

Comment:
can we add docs for the methods like in `campus_eatery.py` for consistency?

Reply:
Added